lib/rubocop/cop/style/bitwise_predicate.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Prefer bitwise predicate methods over direct comparison operations.
+      #
+      # @example
+      #
+      #   # bad - checks any set bits
+      #   (variable & flags).positive?
+      #
+      #   # good
+      #   variable.anybits?(flags)
+      #
+      #   # bad - checks all set bits
+      #   (variable & flags) == flags
+      #
+      #   # good
+      #   variable.allbits?(flags)
+      #
+      #   # bad - checks no set bits
+      #   (variable & flags).zero?
+      #
+      #   # good
+      #   variable.nobits?(flags)
+      #
+      class BitwisePredicate < Base
+        extend AutoCorrector
+        extend TargetRubyVersion
+
+        MSG = 'Replace with `%<preferred>s`.'
```

Comment:
A small thing (and I am not sure whether Rubocop has a guidelines for that — BTW, has it?..), but I think that the message could be:
1. A bit softer in tone
2. A bit more educational (why are we proposing that)

Like
> Prefer `%<preferred>s` for comparison with bit flags.

...or something like that.

Reply:
Style guide proposal https://github.com/rubocop/ruby-style-guide/pull/944 has been merged. But `Prefer %<preferred>s for comparison with bit flags.` is better, I think. Thank you.

Reply:
Cool :+1: 

(When I asked about the guideline, I meant “whether there is some Rubocop guide about the preferred way of offense phrasing”? I believe they aren’t always consistent and sometimes lack the “educational” merit, especially valuable for newer Ruby features. 

Not the fault of cops authors, of course, but maybe some kind of official recommendation about phrasing/level of details might help.

cc @bbatsov)

Reply:
We never wrote anything official, but most cops stick to more or less the same language, so probably we should standardize it at some point and add something here https://docs.rubocop.org/rubocop/1.65/development.html#create-a-new-cop.

Might also be good to have some internal cops checking for the message consistency (see https://github.com/rubocop/rubocop/tree/master/lib/rubocop/cop/internal_affairs)

But I guess that's all a topic for a dedicated conversation/discussion.